cli.js

```diff
@@ -23,4 +30,15 @@ const cli = meow(`
 
 cli.flags.app = cli.input.slice(1);
 
-opn(cli.input[0], cli.flags);
+const getInput = cb => {
+	return cli.input[0] ?
+		cb(cli.input[0]) : getStdin().then(stdin => cb(stdin));
+};
+
+const transform = input => {
+	input = input.trim();
+	return input.slice(0, 5) === 'data:' ?
+		tempWrite.sync(dataUriToBuffer(input)) : input;
```

Comment:
I'm not sure if the absence of file extension will cause any issues, I'll try tomorrow